left_camera:
  pose:
    position: [0.3, 0.2, 1.0]
    orientation: [0.0, 0.0, 0.0, 1.0]
  intrinsics:
    fx: 525.0
    fy: 525.0
    cx: 319.5
    cy: 239.5
  resolution:
    width: 640
    height: 480

right_camera:
  pose:
    position: [0.3, -0.2, 1.0]
    orientation: [0.0, 0.0, 0.0, 1.0]
  intrinsics:
    fx: 525.0
    fy: 525.0
    cx: 319.5
    cy: 239.5
  resolution:
    width: 640
    height: 480
